modules/UpgradeWizard/language/en_us.lang.php

```diff
@@ -177,7 +177,7 @@
     'LBL_UW_SKIPPED_FILES_TITLE' => 'Skipped Files',
     'LBL_UW_SQL_RUN' => 'Check when SQL has been manually run',
     'LBL_UW_START_DESC' => 'This wizard will assist you in upgrading this SuiteCRM instance.',
-    'LBL_UW_START_DESC2' => 'Note: We recommend that you backup the SuiteCRM database and the system files (all of the files in the SuiteCRM folder) before upgrading your production system. We highly recommend that you perform a test upgrade on a cloned instance of your production system first.',
+    'LBL_UW_START_DESC2' => 'Note: We recommend that you backup the SuiteCRM database and the system files (all of the files in the SuiteCRM folder) before upgrading your production system. We highly recommend that you perform a test upgrade on a cloned instance of your production system first. You may also need to run "composer update" if you have customised the composer.json file',
```

Comment:
I think this will scare non tech admin users :smile: Is this the best place to place it or even word it. This message will be BEFORE we have copied anythign over so it really should state that you run the command AFTER the upgrade script yeah?

'We highly recommend that you perform a test upgrade on a cloned instance of your production system first and run a composer update after upgrading if you have a customised composer.json file'

Reply:
@horus68 I get what you are saying, but doesn't that increase the language array?
Whats the impact of changing the description? It does keep things together (though doesn't justify the START_DESC though, something that may require adjusting) especially when its regarding to providing the user with more information.

Reply:
@samus-aran you can apply the change to the value here. I will just remove the approved status on Crowdin so we don't loose translations already done for that string. Translators will be required to verify the translation later.